packages/visx-wordcloud/test/Wordcloud.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { mount } from 'enzyme';
```

Comment:
thanks for writing tests!

just noting that we should likely test this with `react@17`. in that upgrade (#1268) we had to swap a few tests for `react-testing-library` that broke since `enzyme` still doesn't support `react@17`.

cc @gazcn007 do you see anything here you think would be problematic with `react@17`?

Reply:
I think this actually is already maybe failing CI, might be easiest to use `react-testing-library` – hopefully just swapping `mount` calls `render`